pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: browser_launcher
 description: A standardized way to launch web browsers.
 
-version: 0.1.0-dev
+version: 0.1.0
```

Comment:
Should change this to `0.1.1-dev` since since this will go out after 0.1.0 – right?